NeuralAmpModeler/NeuralAmpModelerControls.h

```diff
@@ -293,7 +293,9 @@ class NAMFileBrowserControl : public IDirBrowseControlBase
     auto clearFileFunc = [&](IControl* pCaller) {
       pCaller->GetDelegate()->SendArbitraryMsgFromUI(mClearMsgTag);
       mFileNameControl->SetLabelAndTooltip(mDefaultLabelStr.Get());
-      pCaller->GetUI()->GetControlWithTag(kCtrlTagOutNorm)->SetDisabled(false);
+
+      const bool namActive = pCaller->GetUI()->GetControlWithParamIdx(kNamToggle)->GetValue();
```

Comment:
What if `GetUI()` returns null?

Reply:
Will crash, but same would happen without my changes since `GetControlWithTag` was called without any null-check (as in all other-places in Controls).

Reply:
Interesting point. Let me note this in a separate Issue to follow up on.